### PR TITLE
Lint `newgem` templates

### PR DESCRIPTION
**Pull request by zachahn**
When creating a new gem, one of the first things that I do afterwards is install rubocop and swap the single quotes to double quotes.

I noticed that your `.rubocop.yml` also prefers double quotes, so I was hoping this would be a non-controversial (lol), very slight improvement. I also linted some of the code in ERB tags, but that wasn't my focus.

Some questions I had were

- Should I change the `%q{TODO: ...}` in the gemspec to double quotes as well?
- Should I freeze the `VERSION` constant?
- Does this warrant an entry in the changelog?

Let me know if I should undo anything, or if I should do anything else while I'm here! Thanks 😄

### CODE REVIEW

File: lib/bundler/templates/newgem/lib/newgem.rb.tt [hunk 1]
```diff
@@ -4,9 +4,9 @@ require "<%=config[:namespaced_path]%>/<%=config[:underscored_name]%>"
 <%- end -%>
 
 <%- config[:constant_array].each_with_index do |c,i| -%>
-<%= '  '*i %>module <%= c %>
+<%= "  "*i %>module <%= c %>
```
Comment (segiddins) [inline lib/bundler/templates/newgem/lib/newgem.rb.tt]: Space between operator and operands

**Comment by segiddins** [comment]
Looks pretty great to me, thanks so much!

**Comment by segiddins** [comment]
@bundlerbot r+

**Comment by zachahn** [comment]
Thank you!

**Comment by segiddins** [comment]
Thank _you_!

**Comment by teeparham** [comment]
I just created a new gem & this made my day! 🥇

**Comment by jimbali** [comment]
Le sigh. Rubocop prefers single quotes by default! Dx

**Pull request merged**